library/src/morpheus-master/sass/modules/_footer.scss

```diff
@@ -36,6 +36,11 @@
 	}
 	
 	##{$namespace}footer--poweredBy {
+		a {
+			background-color: var(--footer-PoweredBy-bg);
```

Comment:
What happens when `--footer-PoweredBy-bg` isn't specified by the dark theme?

Reply:
Fails gracefully and has a transparent background color. I can specific a fallback value of this gray if we want to avoid that.